src/share/classes/java/util/ResourceBundle.java

```diff
@@ -57,6 +57,8 @@
 import java.util.jar.JarEntry;
 import java.util.spi.ResourceBundleControlProvider;
 
+import com.alibaba.tenant.TenantContainer;
+import com.alibaba.tenant.TenantGlobals;
 import sun.reflect.CallerSensitive;
 import sun.reflect.Reflection;
 import sun.util.locale.BaseLocale;
```

Comment:
make all import package 'sun.util.*' together

Reply:
no, that is against coding convention and changing irrelevant code